### ISSUE TITLE
Chirp driver not working..

### ISSUE THREAD

**Issue by lohtse** [egzumer/uvk5-chirp-driver#40]
As of 22,03,24 chirp update the driver nolonger works......  Refuses to load up  so any attempt to download up load will fail.  Hope you can sort this soon

**Comment by josephburt** [comment]
nothing with the driver or firmware has been updated since February... I would assume there would be an issue with the device you are using at this time. Could you give more information for someone to further assist. Have you checked to see if your machine has updated. Verified the radio was charged and battery properly seated. Also check the connection of the programming cable to the radio as well as the cable to verify there is no damage there.

**Comment by lohtse** [comment]
> nothing with the driver or firmware has been updated since February... I would assume there would be an issue with the device you are using at this time. Could you give more information for someone to further assist. Have you checked to see if your machine has updated. Verified the radio was charged and battery properly seated. Also check the connection of the programming cable to the radio as well as the cable to verify there is no damage there.

It states Firmware not supported and will not progress further... If I load the driver for IJV firmware it sees it and downloads but the date is corrupted.

If I load an older firmware it works without issues  same if I use and other firmware....No issues. Can download/upload to radio fine... SO that suggests it is not a device,cable,chirp issue.

**Comment by josephburt** [comment]
It could be a corrupt firmware have you tried checking to verify the .bin firmware is good? Which version of chirp are you using? 

I see the last update on the firmware being in the chirp software. Also chirp software being updated on the 22nd I am wondering if there was an issue with the chirp software and the driver or how it uses it?

**Comment by lohtse** [comment]
> It could be a corrupt firmware have you tried checking to verify the .bin firmware is good? Which version of chirp are you using?
> 
> I see the last update on the firmware being in the chirp software. Also chirp software being updated on the 22nd I am wondering if there was an issue with the chirp software and the driver or how it uses it?

Nope..... Firmware is fine..... If I use the 0.81 version of the driver with latest chirp(as stated in the first post) it all works as it should...  Latest version of driver(v1 and onwards) has the issues mentioned..

**Comment by egzumer** [comment]
Firmware not supported means that you are not using my firmware. It has to have firmware string starting with "EGZUMER " (or some other supported firmware). It works fine, I've just checked.

**Comment by lohtse** [comment]
> Firmware not supported means that you are not using my firmware. It has to have firmware string starting with "EGZUMER " (or some other supported firmware). It works fine, I've just checked.

It is yours... and 0.81 works perfectly fine

**Comment by egzumer** [comment]
I've just downloaded the [newest chirp](https://archive.chirpmyradio.com/chirp_next/next-20240322/chirp-next-20240322-win64.zip), it works just fine out of the box by selecting UV-K5.

v1.0 and v0.8.1 are totally different drivers. v0.8.1 doesn't check for the firmware string. 

As I said the error means the firmware version string read from the radio does not start with "EGZUMER "
